docs/source/user/interface.rst

```diff
@@ -116,11 +116,13 @@ The browser’s native context menu can be accessed by holding down
 Keyboard Shortcuts
 ~~~~~~~~~~~~~~~~~~
 
-As in the classic Notebook, you can navigate the user interface through keyboard shortcuts.
-To find these shortcuts, go to the File, select the Settings menu option, then select Keyboard Shortcuts.
-With Keyboard Shortcuts in the settings editor, you can see a full list of keyboard shortcuts, and customize them as well.
+As in the classic Notebook, you can navigate the user interface through keyboard
+shortcuts. You can find and customize the current list of keyboard shortcuts by
+selecting the Advanced Settings Editor item in the Settings menu, then selecting
+Keyboard Shortcuts in the settings tab.
```

Comment:
Are we doing settings with a capital S since it's the label of the tab?

Reply:
Sure - we are helping them find the specific tab, so that makes sense to me.

Reply:
+1